security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -92,8 +92,8 @@
 }
 
 /* define alert log limit */
-if (!empty($config['installedpackages']['snortglobal']['alert_log_limit_size']) && $config['installedpackages']['snortglobal']['alert_log_limit_size'] != "0")
-	$alert_log_limit_size = $config['installedpackages']['snortglobal']['alert_log_limit_size'] . "K";
+if (!empty(config_get_path('installedpackages/snortglobal/alert_log_limit_size')) && config_get_path('installedpackages/snortglobal/alert_log_limit_size') != "0")
```

Comment:
```suggestion
if (config_get_path('installedpackages/snortglobal/alert_log_limit_size', '0') != '0')
```

Reply:
The behavior of config_get_path() and array_get_path() is to return $default if it is non-null and the key does not exist or its value is an empty string. This seemed to be a common enough idiom to be worth making simpler with the config interface.

Reply:
Thanks for the tip. Adjusted all of the marked code sections to take advantage of a default return value.